formatter.go

```diff
@@ -30,7 +38,7 @@ type Formatter interface {
 //
 // It's not exported because it's still using Data in an opinionated way. It's to
 // avoid code duplication between the two default formatters.
-func prefixFieldClashes(data Fields) {
+func prefixFieldClashes(data Fields, hasRemoteCaller bool) {
```

Comment:
It probably is. Though, In the progression of the branch ([here](https://github.com/victorhaggqvist/logrus/tree/feat/remote-caller-use)), I have actually removed the prefixing entirely, in favour of a more specific name ie. `remote_caller`.